tools/omero/omero_dataset_to_plate.xml

```diff
@@ -0,0 +1,67 @@
+<tool id="omero_dataset_to_plate" name="OMERO Dataset to Plate" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@" license="MIT">
+   <description> with omero-py </description>
+   <macros>
+       <import>macros.xml</import>
+       <token name="@VERSION_SUFFIX@">3</token>
+   </macros>
+   <xrefs>
+       <xref type="bio.tools">omero</xref>
+   </xrefs>
+   <expand macro="omeropy_requirements"/>
+   <command detect_errors="exit_code"><![CDATA[
+       python '$__tool_directory__/omero_dataset_to_plate.py'
+       --credential-file '$credentials'
+       @HOST_PORT@
+       --dataset_id '$dataset_id'
+       --log_file '$log'
+       --mapping_file '$mapping'
+       --delete_dataset '$delete_dataset'
+   ]]></command>
+   <configfiles>
+       <expand macro="credentials"/>
+   </configfiles>
+   <inputs>
+       <expand macro="host_port"/>
+       <param name="dataset_id" type="integer" optional="false" label="Dataset ID to convert to a plate"/>
```

Comment:
Do ids have a min?

Reply:
Do you mean a min value?

Reply:
Yes

Reply:
They start from 1 and can only increase, should we add a validator?

Reply:
You can just add  `min="1"` to the `param`.

Reply:
ok, corrected for all the xml tools requiring and id as input :)